projects/angular/src/wizard/_wizard.clarity.scss

```diff
@@ -332,6 +332,8 @@
     .clr-wizard-stepnav-link {
       width: 100%;
       display: flex;
+      place-content: normal;
+      gap: 0;
```

Comment:
Probably won't needs these with the link change reverted.

Reply:
corrected in commit bf22d26